<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TunerRegister</class>
 <widget class="QWidget" name="TunerRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>45</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="tunerAddr">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tunerHex">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tunerBin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>13</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Addr</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>13</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Hex</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>13</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Bin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tunerGet">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>10</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Get</string>
   </property>
  </widget>
  <widget class="QPushButton" name="tunerSet">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>79</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tunerResult">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
